### ISSUE TITLE
The latest release does not start language server configured in coc-settings.json

### ISSUE THREAD

**Issue by FongHou** [neoclide/coc.nvim#1878]
## versions

  vim version: NVIM v0.5.0-486-gd13c1642a
  node version: v14.2.0
  coc.nvim version: 0.0.78-fd9e7d3972
  term: tmux-256color
  platform: linux

  ## Output channel: git

  Looking for git in: git
  > git rev-parse --show-toplevel
  resolved root: /mnt/c/Users/houfe/haskeller/dev/yesod-app
  resolved root: /mnt/c/Users/houfe/haskeller/dev/yesod-app
  > git symbolic-ref --short HEAD
  > git diff --name-status
  > git diff --staged --name-status
  > git ls-files --others --exclude-standard
  resolved root: /mnt/c/Users/houfe/haskeller/dev/yesod-app
  resolved root: /mnt/c/Users/houfe/haskeller/dev/yesod-app
  > git symbolic-ref --short HEAD
  > git diff --name-status
  > git diff --staged --name-status
  > git ls-files --others --exclude-standard
  resolved root: /mnt/c/Users/houfe/haskeller/dev/yesod-app
  > git ls-files -- src/Api/User.hs
  resolved root: /mnt/c/Users/houfe/haskeller/dev/yesod-app
  > git symbolic-ref --short HEAD
  > git --no-pager show :src/Api/User.hs
  > git diff --name-status
  > git diff --staged --name-status
  > git ls-files --others --exclude-standard
  > git diff /mnt/c/Users/houfe/haskeller/dev/yesod-app/src/Api/User.hs
  resolved root: /mnt/c/Users/houfe/haskeller/dev/yesod-app
  resolved root: /mnt/c/Users/houfe/haskeller/dev/yesod-app
  > git symbolic-ref --short HEAD
  > git ls-files -- src/Api/User.hs
  > git diff --name-status
  > git diff --staged --name-status
  > git ls-files --others --exclude-standard
  > git --no-pager show :src/Api/User.hs
  > git diff /mnt/c/Users/houfe/haskeller/dev/yesod-app/src/Api/User.hs

  ## Output channel: prettier

  ## Output channel: snippets

  [Info 3:27:17 PM] Using ultisnips directories: UltiSnips /mnt/c/Users/houfe/haskeller/.config/coc/ultisnips
  [Info 3:27:17 PM] Using ultisnips python command: pyx

## Describe the bug

Latest release does not start any languageserver configured in coc-settings.json. 

## Reproduce the bug

Reproduce coc-settings.json. It seems none of external language server starts anymore after the last release.

  "languageserver": {
    "ghcide": {
      "command": "ghcide",
      "args": ["--lsp"],
      "rootPatterns": [
        "stack.yaml",
        ".hie-bios",
        "cabal.config",
        "package.yaml"
      ],
      "filetypes": ["hs", "lhs", "haskell"],
      "initializationOptions": {
        "languageServerHaskell": {
          "hlintOn": true,
          "maxNumberOfProblems": 10,
          "completionSnippetsOn": true
        }
      }
    },
    "dhall": {
      "command": "dhall-lsp-server",
      "filetypes": ["dhall"],
      "initializationOptions": {},
      "settings": {}
    }
  },

**Comment by dpriedel** [comment]
+1.  I noticed this after updating with release version today.

**Comment by 0xA1F** [comment]
Can confirm this issue as well.

@dpriedel @FongHou I've managed to overcome this behavior by removing release version and installing latest version via `Plug 'neoclide/coc.nvim', {'do': 'yarn install --frozen-lockfile'}
`

**Comment by oblitum** [comment]
No need to install from source though, just pick previous commit, vim-plug support commit refs.

**Comment by FongHou** [comment]
yes, I saw master merged a potential fix.

Here is what I did to my local release branch. Essentially just rollback the massive "tslint to eslint" commit.

```
git fetch --unshallow
git reset --hard 244b439
```

**Comment by fannheyward** [comment]
Fixed on master, release branch needs to upgrade, cc @chemzqm .

**Comment by npearson72** [comment]
This did not fix https://github.com/neoclide/coc.nvim/issues/1881 yet @oblitum closed it as a dupe. Am I missing something?

**Comment by cigrainger** [comment]
Yep I've wasted a bunch of time today figuring out why this isn't working. Anything I can do to help?

**Comment by npearson72** [comment]
> Yep I've wasted a bunch of time today figuring out why this isn't working. Anything I can do to help?

Sorry, are you asking me something?

**Comment by chemzqm** [comment]
Fixed, we will add tests to prevent this kind of issue.

**Comment by npearson72** [comment]
@chemzqm I think there's a miscommunication. I'm referring to #1881 

Still an issue:

```
➜ tree coc
coc
├── extensions
│   ├── db.json
│   └── package.json
└── memos.json

1 directory, 3 files
```

**Comment by GiuseppeCesarano** [comment]
I don't know if it's only for me, but ccls still doesn't work. I'm using the suggested configuration in the wiki

**Comment by oblitum** [comment]
> This did not fix #1881 yet @oblitum closed it as a dupe. Am I missing something?

@npearson72, etc 

I closed #1881 as duplicate of #1878 because it looked like same report, not because it any was solved.

This one (#1878) was solved by pull #1882, which claimed to fix both #1878 and #1881.

This one (#1878) was closed because solution was already on master, but not on release branch. So if one wanted to test the solution would have to pull and compile from master.

It's now already on release branch and it's expected to be fixed after any update.
